test/e2e.sh

```diff
@@ -0,0 +1,98 @@
+#! /bin/bash
+
+set -u -o pipefail 
+
+# Globals
+test_ns="driver-toolkit-test"
+dtk_registry_image="image-registry.openshift-image-registry.svc:5000/openshift/driver-toolkit:latest"
+
+# Get running RHCOS version
+# TODO check RT kernel version
+find_rhcos_version() {
+    worker_node=$(kubectl get nodes --selector='node-role.kubernetes.io/worker' -ojsonpath='{.items[].metadata.name}')
+    etc_os_release=$(oc debug node/${worker_node} \
+	    --quiet \
+	    -- cat /host/etc/os-release \
+	    | grep OSTREE_VERSION)
+
+    eval $(echo ${etc_os_release})
```

Comment:
hum, interesting `eval`, I guess it's worth a comment so that people don't have to read twice or thrice to understand what this line does ;-)

I would think that 
```   ...
- cat /host/etc/os-release \
	    | grep OSTREE_VERSION \
            | cut -d= -f2 
)
```
is a safer design, just to avoid `eval` from "a random server" (🙃),  but the risk is limited here :)

Reply:
I hadn't thought of the risk of this eval... To make it more clear what's going on I took your suggestion, but also added `tr -d \'\"` because of the mixed use of quotes in the rhcos /etc/os-release file.